workers/unity/Assets/Config/GdkToolsConfiguration.json

```diff
@@ -4,5 +4,6 @@
         "../../schema"
     ],
     "CodegenOutputDir": "Assets/Generated/Source",
-    "RuntimeIp": ""
-}
+    "RuntimeIp": "",
+    "DevAuthTokenDir": "Resources"
```

Comment:
So if we store this here - I'm not sure if we can actually read this file at Runtime as we don't compile the `Tools` asmdef into the binaries (which contain the `MiniJSON` tools)

Reply:
Spoke with Jess offline: if we restrict DATs to always be at the root of a Resources folder (dir ends in `/Resources` or equals `Resources`), then we can just load directly without worrying about the path inside `Resources`

Reply:
If this is restricted to `Resources`, why is it configurable at all?

Reply:
because the `Resources` folder could be anywhere in your project or they might have multiple